SLHCUpgradeSimulations/Geometry/test/Phase2PixelNtuple.cc

```diff
@@ -108,11 +127,18 @@ class Phase2PixelNtuple : public edm::one::EDAnalyzer<> {
   edm::ParameterSet const conf_;
   TrackerHitAssociator::Config trackerHitAssociatorConfig_;
   edm::EDGetTokenT<edmNew::DetSetVector<SiPixelRecHit>> pixelRecHits_token;
+  std::string ttrhBuilder_;
   edm::EDGetTokenT<edm::View<reco::Track>> token_recoTracks;
+  edm::EDGetTokenT<TrajTrackAssociationCollection> TTAToken;
```

Comment:
member variables should be named consistently to differentiate from local variables, e.g. always ending with underscore. they should also start with lowercase. (http://cms-sw.github.io/cms_coding_rules.html 2.9)

Reply:
Sorry, I have now fixed this (and other variables) in the latest commit